app/services/candidate_interface/continuous_applications/inactive_date_calculator.rb

```diff
@@ -0,0 +1,36 @@
+module CandidateInterface
+  module ContinuousApplications
+    class InactiveDateCalculator
+      def initialize(application_choice:, effective_date:, time_limit_calculator: TimeLimitCalculator)
```

Comment:
Does it make sense to default `effective_date` to `Time.zone.now`?

Reply:
Well, it does makes sense but from the Submit aplication PoV (that's why I put it there). This class will be "give you pass an effective_date I will calculate the inactive date for you".

Reply:
Right, I just imagined that most of the time you'll want to make that calculation from the moment you call the class, like in the Submit class

```
 alias effective_date current_time
```